projects/packages/forms/src/blocks/field-rating/edit.js

```diff
@@ -82,37 +76,41 @@ export default function RatingFieldEdit( props ) {
 				<div { ...innerBlocksProps } />
 			</BlockContextProvider>
 
-			<InspectorControls>
-				<PanelBody title={ __( 'Settings', 'jetpack-forms' ) }>
-					<NumberControl
-						__next40pxDefaultSize
-						__unstableInputWidth="50%"
-						help={ __( 'Highest rating users can select (2–10).', 'jetpack-forms' ) }
-						label={ __( 'Maximum rating', 'jetpack-forms' ) }
-						max={ 10 }
-						min={ 2 }
-						onChange={ updateMax }
-						spinControls="custom"
-						value={ max }
-					/>
-					<RangeControl
-						label={ __( 'Default rating', 'jetpack-forms' ) }
-						help={ __( 'Pre-selected rating value (0 for no selection)', 'jetpack-forms' ) }
-						min={ 0 }
-						max={ max }
-						value={ defaultValue }
-						onChange={ onChangeDefault }
-					/>
-				</PanelBody>
-			</InspectorControls>
-
 			<JetpackFieldControls
 				clientId={ clientId }
 				id={ id }
 				required={ required }
 				attributes={ attributes }
 				setAttributes={ setAttributes }
 				width={ width }
+				extraFieldSettings={ [
+					{
+						index: 1,
+						element: (
+							<>
+								<NumberControl
+									__next40pxDefaultSize
+									__unstableInputWidth="50%"
+									help={ __( 'Highest rating users can select (2–10).', 'jetpack-forms' ) }
+									label={ __( 'Maximum rating', 'jetpack-forms' ) }
+									max={ 10 }
+									min={ 2 }
+									onChange={ updateMax }
+									spinControls="custom"
+									value={ max }
+								/>
+								<RangeControl
+									label={ __( 'Default rating', 'jetpack-forms' ) }
+									help={ __( 'Pre-selected rating value (0 for no selection)', 'jetpack-forms' ) }
```

Comment:
Based on the screenshot. This creates an orphan. 

Maybe we could try one of these: 
```
Default rating (0 = none selected)
Starting rating value (0 means no rating chosen)
Initial rating (0 = no selection)
Pre-filled rating (set to 0 if not selected)
Default value (0 = no rating selected)
Auto-set rating (0 indicates no choice yet)
```